lib/cpp/test/SecurityTest.cpp

```diff
@@ -75,13 +75,17 @@ struct GlobalFixture
             if (!boost::filesystem::exists(certFile("server.crt")))
             {
                 keyDir = boost::filesystem::path(master_test_suite().argv[master_test_suite().argc - 1]);
-                BOOST_REQUIRE_MESSAGE(boost::filesystem::exists(certFile("server.crt")),
-                                      "The last argument to this test must be the directory containing the test certificate(s).");
+                if (!boost::filesystem::exists(certFile("server.crt")))
+                {
+                	std::cerr << "The last argument to this test must be the directory containing the test certificate(s)." << std::endl;
```

Comment:
Starting with boost-1.59 you cannot use assertion test macros outside a test scope.  This means you cannot use them in the global fixture.  It's a weird choice but that's why I made this change.